DSharpPlus.Commands/CommandsExtension.cs

```diff
@@ -411,12 +411,24 @@ private static async Task DefaultCommandErrorHandlerAsync(CommandsExtension exte
-            ChecksFailedException checksFailedException => $"The following context checks failed: {Formatter.InlineCode(Formatter.Sanitize(string.Join("\n\n", checksFailedException.Errors.Select(x => x.ErrorMessage))))}.",
-            DiscordException discordException when discordException.Response is not null && (int)discordException.Response.StatusCode >= 500 && (int)discordException.Response.StatusCode < 600 => $"Discord API error {discordException.Response.StatusCode} occurred: {discordException.JsonMessage ?? "No further information was provided."}",
-            DiscordException discordException when discordException.Response is not null => $"Discord API error {discordException.Response.StatusCode} occurred: {discordException.JsonMessage ?? discordException.Message}",
+            CommandNotFoundException commandNotFoundException 
+                => $"Command {Formatter.InlineCode(Formatter.Sanitize(commandNotFoundException.CommandName))} was not found.",
+            ArgumentParseException argumentParseException 
+                => $"Failed to parse argument {Formatter.InlineCode(Formatter.Sanitize(argumentParseException.Parameter.Name))}.",
+            ChecksFailedException checksFailedException when checksFailedException.Errors.Count == 1 
+                => $"The following error occurred: {Formatter.InlineCode(Formatter.Sanitize(checksFailedException.Errors[0].ErrorMessage))}",
+            ChecksFailedException checksFailedException 
+                => $"The following context checks failed: {Formatter.InlineCode(Formatter.Sanitize(string.Join("\n\n", checksFailedException.Errors.Select(x => x.ErrorMessage))))}.",
+            ParameterChecksFailedException checksFailedException when checksFailedException.Errors.Count == 1
+                => $"The following error occurred: {Formatter.InlineCode(Formatter.Sanitize(checksFailedException.Errors[0].ErrorMessage))}",
+            ParameterChecksFailedException checksFailedException
+                => $"The following context checks failed: {Formatter.InlineCode(Formatter.Sanitize(string.Join("\n\n", checksFailedException.Errors.Select(x => x.ErrorMessage))))}.",
+            DiscordException discordException when discordException.Response is not null 
+                && (int)discordException.Response.StatusCode >= 500 
+                && (int)discordException.Response.StatusCode < 600 
+                => $"Discord API error {discordException.Response.StatusCode} occurred: {discordException.JsonMessage ?? "No further information was provided."}",
+            DiscordException discordException when discordException.Response is not null 
+            => $"Discord API error {discordException.Response.StatusCode} occurred: {discordException.JsonMessage ?? discordException.Message}",

```

Comment:
Intentional diff?

Reply:
no, but judging by the fact that diff is already merged it won't apply